modules/sso-rbac/provisioner.tf

```diff
@@ -0,0 +1,20 @@
+data "aws_iam_roles" "permission_set_arns" {
+  for_each    = { for kr in var.bindings : "${kr.namespace}-${kr.group}" => kr }
+  path_prefix = "/aws-reserved/sso.amazonaws.com/eu-west-1/AWSReservedSSO_permission-set-${each.value.group}"
+}
+
+data "aws_iam_roles" "sso" {
```

Comment:
this can stay and you bring here also locals{} block from above external github sourced module